frontend/src/metabase/browse/components/ModelsTable.module.css

```
@@ -0,0 +1,10 @@
+.collectionLink {
+  :hover {
+    &,
+    *,
+    *:before,
+    *:after {
+      color: var(--mb-color-brand) !important;
+    }
+  }
+}
```

this seems to work just fine? what does all that extra stuff do?

```suggestion
.collectionLink {
  :hover {
    color: var(--mb-color-brand);
  }
}
```